linetools/spectra/lsf.py

```diff
@@ -450,7 +458,13 @@ def interpolate_to_wv0(self, wv0):
         lsf_vals = Column(name='kernel', data=lsf_vals)
 
         # create column of relative pixel in absolute wavelength
-        wv_array = [(self.pixel_scale * self._data['rel_pix'][i] + wv0*u.AA).value for i in range(len(self._data))]
+        if not isinstance(self.pixel_scale,u.quantity.Quantity):
+            # deal with wavelength-dependent pixel scale (i.e., STIS echelle)
+            scalefac = 1./float(self.pixel_scale.split('/')[-1])
+            wv_array = [(wv0*u.AA * (1. + scalefac * self._data['rel_pix'][i])).value
+                        for i in range(len(self._data))]
```

Comment:
There could be consequences when applying this approach to spectra where delta_wavelength is not constant for VP fitting...